pcmueller/Hookshot
# Feature/ux functionality

## Is this a fix or a feature?
Feature

## What is the change or fix?
- Completed "location" dropdown menu functionality - user can now select location and store `currentLocation` in local state.
- Set up basic Router functionality, using Switch to render either `Entry` or `Main` component page view.
- Configured Router to use dropdown / "enter" button event data to render appropriate URL, passing through location data.
- Use location data passed into Main to display "welcome" message to user, specifically referencing selected location.
- Reformatted and restyled Entry and Main CSS elements.
- Prevent user from clicking "enter" button / routing to Main page until "location" selection has been made in dropdown.

## How should this be tested?
- Display in browser, check React dev tools for `this.state.currentLocation` data persistence after page reroute / refresh.

## What are the next steps?
- Complete "category button" functionality.  Data currently fetches on each button click, so next up:
-- Build out "ResultsPage" component structure and basic HTML/CSS layout and styling.
-- Connect API data and render to DOM
-- Connect MainPage to ResultsPage using Router.
